drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -408,6 +433,24 @@ def _get_item_definition(self, item_id):
         """
         return next(i for i in self.data['items'] if i['id'] == item_id)
 
+    def _get_valid_item_zones(self, item_id):
+        """
+        Returns a list of the zones that are valid options for the item. If
+        we have an existing configured list of options, return that. If we
+        have an existing single item, return that encapsulated in a list.
+        If we don't have either, which would be weird, return an empty list.
+        """
+        item = self._get_item_definition(item_id)
+
+        if item.get('zones') is not None:
+            return item.get('zones')
+
+        elif item.get('zone') is not None and item.get('zone') != 'none':
+            return [item.get('zone')]
+
+        else:
+            return []
```

Comment:
@haikuginger Nit: This code is nice and readable -- no need for all that vertical whitespace :)